includes/controller/api.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Creates a link or url to a given resource.
+ *
+ * @param $resource
+ */
+function api_link($resource)
+{
+    return page_link_to('api') . '&r=' . $resource;
+}
+
+function getApiShifts()
+{
+    $dbResult = sql_select(
+        "SELECT s.*, r.Name as locationName, r.location, r.lat, r.long, t.name as shiftType FROM `Shifts` s INNER JOIN `ShiftTypes` t ON s.`shifttype_id` = t.`id` INNER JOIN `Room` r ON r.`RID` = s.`RID` GROUP BY s.`SID` ORDER BY s.`start`"
```

Comment:
I see a bigger problem in the security: As the dashboard uses the api from outside the firewall (no auth) we should not pass private data.